features/step_definitions/journey/registration_steps.rb

```diff
@@ -124,14 +124,14 @@
   puts "Registration #{@reg_number} seeded"
 end
 
-Given("I create a new registration as {string}") do |account_email|
+Given("I create a new registration") do
   load_all_apps
   @tier = :upper
   @organisation_type = :limitedCompany
-  seed_data = SeedData.new("limitedCompany_complete_active_registration.json", "accountEmail" => account_email)
+  seed_data = SeedData.new("limitedCompany_complete_active_registration.json",
+                           "expires_on" => (DateTime.now + 30).to_s)
```

Comment:
Expiry date doesn't need to be set. There is a step `I have an active registration` that could be used instead.

Reply:
I've removed the unnecessary setting of expiry date throughout this file.